### PR TITLE
Allow plugins to be started & stopped at run time through the PluginM…

### PR DESCRIPTION
**Pull request by nomis52**
…anager

### CODE REVIEW

**Comment by nomis52** [comment]
FYI: @daveol 

Now you can remove the call to reload.

File: olad/PluginManager.cpp [hunk 1]
```diff
@@ -178,4 +181,54 @@ void PluginManager::GetConflictList(ola_plugin_id plugin_id,
...
+    OLA_INFO << "Started " << plugin->Name();
+    STLReplace(&m_active_plugins, plugin->Id(), plugin);
+  }
+  return ok;
+}
+
+/*
+ * @brief Check if this plugin conflicts with any of the running plugins.
+ * @param plugin The plugin to check
+ * @returns The first conflicting plugin, or NULL if there isn't one.
```
Comment (peternewman) [inline olad/PluginManager.cpp]: SPaG aren't any

**Comment by peternewman** [comment]
LGTM.
I'm sure you told me not to do this before... :)

**Comment by nomis52** [comment]
Yeah, I didn't realize the complexity until I started testing Dave's changes.

**Pull request merged**